src/components/home/home.tsx

```diff
@@ -218,7 +220,19 @@ const Home = () => {
             </Carousel.Item>
           </Carousel>
         </Col>
-        <Col className="rounded">Twitter</Col>
+        <Col className="rounded">
+          <TwitterTimelineEmbed
+            sourceType="profile"
+            screenName="GHGA_DE"
+            options={{ height: 425 }}
+            placeholder={
+              <p>
+                <Spinner animation="border" size="sm" variant="info" />&nbsp;Loading <a href="https://twitter.com/GHGA_DE">GHGA Twitter timeline</a> ...<br />
+                You may need to disable blocking of third-party cookies for this element to display correctly.
+              </p>
+            }
+          />
+        </Col>
```

Comment:
Are the embed and view in Twitter links part of the component? Can they be removed?

Reply:
Yeah it should be possible. Footer: false. Let me do that quickly.